### PR TITLE
Add EnableTimetracking option to app settings

### PR DESCRIPTION
**Pull request by Bwko**
Add an option to completely disable the time tracking feature.

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

File: modules/setting/setting.go [hunk 1]
```diff
@@ -1172,6 +1173,7 @@ func newService() {
 	Service.EnableCaptcha = sec.Key("ENABLE_CAPTCHA").MustBool()
 	Service.DefaultKeepEmailPrivate = sec.Key("DEFAULT_KEEP_EMAIL_PRIVATE").MustBool()
 	Service.DefaultAllowCreateOrganization = sec.Key("DEFAULT_ALLOW_CREATE_ORGANIZATION").MustBool(true)
+	Service.EnableTimetracking = sec.Key("ENABLE_TIMETRACKING").MustBool(true)
 	Service.DefaultEnableTimetracking = sec.Key("DEFAULT_ENABLE_TIMETRACKING").MustBool(true)
```
Comment (lafriks) [inline modules/setting/setting.go]: Shouldn't default be false if timetracking is disabled in server?

**Review by lafriks — CHANGES_REQUESTED** [review]
Also setting in repository settings should be hidden if global setting is set to false. Also would be nice if this would be added to admin server option list in UI

**Comment by Bwko** [comment]
@lafriks I've updated my PR.

**Comment by jonasfranz** [comment]
@Bwko What is the default value of this option?

**Comment by Bwko** [comment]
The time tracking is enabled by default

**Review by lafriks — APPROVED** [review]

**Pull request merged**